Heroes Handbook, Main File.txt

```diff
@@ -837,76 +837,90 @@ As a lightforged draenei, you've commited yourself to a crusade against the burn
 
-***Superior Darkvision.*** Accustomed to the black nights, you have superior vision in dark and dim conditions. You can see in dim light within 120 feet of you as if it were bright light, and in darkness as if it were dim light. You can't discern color in darkness, only shades of gray.
+***Bite (Worgen form).*** Your fanged maw is a natural weapon, which you can use to make unarmed strikes. If you hit with it, you deal piercing damage equal to 1d6 + your Strength modifier.
 
-***Language of Wolves.*** You are able to communicate with wolves, as the spell *commune with animals*.
+***Hunter's Lore.*** You gain proficiency with one of the following skills of your choice: Animal Handling, Nature, Perception, Stealth, or Survival.
 
-***Natural Weapons.*** Your claws and bite are natural weapons, which you can use to make unarmed strikes. If you hit with your claws, you deal 1d4 + your Strength modifier slashing damage, and with your bite, you deal 1d6 + your Strength modifier piercing damage.
+***Two Forms.*** Using 1 minute, you can transform yourself into a human. While human, you appear as you did before getting affected by the worgen curse. You always appear as the same human when you transform. You can transform back into a worgen as a free action on your turn.
 
-***Truesilver Vulnerability.*** You are vulnerable to damage taken by silvered weapons.
+As a human, your speed is reduced by 5 feet, and you loose any traits marked (Worgen Form). If you make an attack, cast a spell that affects an enemy creature, or try to use an ability only available in worgen form, you imme-diately revert into your worgen form.
 
-***Viciousness.*** When you score a critical hit with a melee attack, you can roll one of the weapon's damage dice one additional time and add it to the extra damage of the critical hit.
+***Viciousness (Worgen form).*** You can throw yourself into a vicious frenzy. As a bonus action, you can make a special attack with your bite. If the attack hits, it deals its normal damage, and you gain temporary hit points (minimum of 1) equal to your Constitution modifier, and you can't use this trait again until you finish a short or long rest.
 
 ***Languages.*** You can speak, read, and write Common and one extra language of your choice. Humans typically learn the languages of other peoples they deal with, including obscure dialects.
 
 <div class='footnote'>PART 1 | RACES</div>
 <img src='https://www.gmbinder.com/images/ymeA1sL.jpg' style='position:absolute; top:0px; right:200px; width:750px' />
 <img src='https://www.gmbinder.com/images/ohZghLL.png' style='position:absolute; top:0px; right:-100px; width:900px' />
-<img src='https://www.gmbinder.com/images/3RBiFq9.png' style='position:absolute; top:640px; right:30px; width:370px' />
+<img src='https://www.gmbinder.com/images/3RBiFq9.png' style='position:absolute; top:960px; right:30px; width:370px' />
```

Comment:
```suggestion
<img src='https://www.gmbinder.com/images/3RBiFq9.png' style='position:absolute; top:650px; right:418px; width:370px' />
```
What if we put the Worgen crest on top of the picture of Greymane, so that it's fully visible on the page?